### PR TITLE
compose ps do list containers, not services

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
Change `compose ps` semantic so it lists containers, not services.
Original semantic could be re-introduced by some `docker service ls` command if we need it

local output:
```
docker --context LOCAL compose ps
NAME                          SERVICE             STATE               PORTS
example-voting-app_db_1       db                  running             5432/tcp
example-voting-app_redis_1    redis               running             0.0.0.0:32768->6379/tcp
example-voting-app_result_1   result              running             0.0.0.0:5001->5001/tcp, 0.0.0.0:5858->5858/tcp
example-voting-app_vote_1     vote                running             0.0.0.0:5000->5000/tcp
example-voting-app_worker_1   worker              running      
```

ecs output:
```
docker --context ECS compose ps
NAME                                                       SERVICE STATE               PORTS
task/example-voting-app/23250671a247442a83fabea30f4a11e3   db      RUNNING             
task/example-voting-app/66bdbc7e700841e995f5d5635946324a   redis   RUNNING             examp-LoadB-PO1UVTDIV58G-197987ae58c1b97c.elb.us-east-1.amazonaws.com:6379->6379/tcp
task/example-voting-app/6afb8ff1d96444c1b07e38b720624622   vote    RUNNING             examp-LoadB-PO1UVTDIV58G-197987ae58c1b97c.elb.us-east-1.amazonaws.com:5000->5000/tcp
task/example-voting-app/75a0aa62202d473585ca4bb66dfa502d   result  RUNNING             examp-LoadB-PO1UVTDIV58G-197987ae58c1b97c.elb.us-east-1.amazonaws.com:5858->5858/tcp, examp-LoadB-PO1UVTDIV58G-197987ae58c1b97c.elb.us-east-1.amazonaws.com:5001->5001/tcp
```

aci output:
```
 docker --context ACI compose ps
NAME                      SERVICE  STATE               PORTS
example-voting-app_redis  redis    Running             20.62.215.21:6379->6379/tcp->6379/
example-voting-app_test   test     Running        
```

**Related issue**
https://github.com/docker/compose-cli/issues/966

/test-aci
/test-ecs

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**
image

### CODE REVIEW

File: api/compose/api.go [hunk 1]
```diff
@@ -56,6 +56,16 @@ type PortPublisher struct {
...
 }
 
+// ContainerSummary hold high-level description of a container
+type ContainerSummary struct {
+	ID         string
+	Name       string
+	Project    string
+	Service    string
+	State      string
+	Publishers []PortPublisher
```
Comment (rumpl) [inline api/compose/api.go:66]: Curious why this isn't `[]Port` as with the container

File: cli/cmd/compose/ps.go [hunk 2]
```diff
@@ -54,44 +54,34 @@ func runPs(ctx context.Context, opts composeOptions) error {
...
+	})
+
+	return formatter.Print(containers, opts.Format, os.Stdout,
 		func(w io.Writer) {
-			for _, service := range view {
-				_, _ = fmt.Fprintf(w, "%s\t%s\t%d/%d\t%s\n", service.ID, service.Name, service.Replicas, service.Desired, strings.Join(service.Ports, ", "))
+			for _, container := range containers {
+				var ports []string
+				for _, p := range container.Publishers {
+					if p.URL == "" {
```
Comment (rumpl) [inline cli/cmd/compose/ps.go:77]: Feel like this isn't right, published ports can be more complicated than just target port -> protocol no?

File: api/compose/api.go (hunk 1 above)
Comment (ndeloof) [reply api/compose/api.go:66]: same as port but includes an `URL`, as cloud integration has to provide external access domain name

**Review by rumpl — APPROVED** [review]

File: cli/cmd/compose/ps.go (hunk 2 above)
Comment (ndeloof) [reply cli/cmd/compose/ps.go:77]: published port is target/protocol exposed _one way or the other_ by infrastructure as published URL

File: api/compose/api.go (hunk 1 above)
Comment (rumpl) [reply api/compose/api.go:66]: Couldn't we use the `HostIP` for that?

Comment (rumpl) [reply api/compose/api.go:66]: I mean yeah sure, it's not an IP but a URL, ok ok ok

File: tests/aci-e2e/e2e-aci_test.go [hunk 3]
```diff
@@ -734,20 +734,23 @@ func TestUpUpdate(t *testing.T) {
...
 		for _, line := range l {
 			fields := strings.Fields(line)
-			containerID := fields[0]
-			switch containerID {
+			name := fields[0]
+			switch name {
 			case wordsContainer:
 				wordsDisplayed = true
-				assert.DeepEqual(t, fields, []string{containerID, "words", "1/1"})
+				assert.Equal(t, fields[1], "Running")
```
Comment (gtardif) [inline tests/aci-e2e/e2e-aci_test.go:741]: that must be fields[2] I guess... :) (containerID, service, status)

**Review by gtardif — APPROVED** [review]
LGTM with fixed red test

**Pull request merged**